cmd/beaker/executor.go

```diff
@@ -30,6 +30,9 @@ type executorConfig struct {
 	// to an NFS mount to enable roaming profiles. If unset, sessions mount the
 	// invoking user's home directory.
 	SessionHome string `yaml:"sessionHome"`
+
+	// (optional) List of allowed mount paths for this machine.
```

Comment:
```suggestion
	// (optional) List of paths that are mounted in the container by default
```

Reply:
This suggestion isn't quite correct since these paths aren't mounted into batch jobs by default.  The following would work better.

```suggestion
	// (optional) List of allowed host mount paths for this machine.
	// All paths will be mounted into interactive sessions by default.
```

Reply:
I'm working on removing the need for this config in the CLI entirely as part of https://github.com/allenai/beaker-service/issues/1623 so I'm going to leave this as-is.